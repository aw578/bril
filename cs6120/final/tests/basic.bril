@main {
  v1: int = const 1;
  v2: int = const 1;
  v3: int = add v1 v2;
  v4: int = add v1 v2;
}