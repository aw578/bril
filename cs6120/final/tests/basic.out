Congruence classes for function main:
Class const 1: ['v1.0', 'v2.0']
Class add: ['v3.0', 'v4.0']
